config/config.go

```diff
@@ -30,6 +32,8 @@ func Get() (*Configuration, error) {
 		GracefulShutdownTimeout:    time.Second * 5,
 		HealthCheckInterval:        30 * time.Second,
 		HealthCheckCriticalTimeout: 90 * time.Second,
+		DefaultLimit:               100,
```

Comment:
Should we be setting a high default, so that existing functionality is not changed? If someone is using the API and expecting the full set of data, once this change is released they will be limited to 100 results?